unity-renderer/Assets/Desktop/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUDDesktop/Scripts/ScreenResolutionUtils.cs

```diff
@@ -7,7 +7,29 @@ namespace MainScripts.DCL.Controllers.HUD.SettingsPanelHUDDesktop.Scripts
     public static class ScreenResolutionUtils
     {
         private static Resolution[] resolutions;
-        public static IReadOnlyList<Resolution> Resolutions => resolutions ??= Screen.resolutions;
+
+        // this offset was made for the 0 index to be Full HD instead of MAX
+        private static int defaultIndex;
+        public static IReadOnlyList<Resolution> Resolutions => resolutions ??= GetResolutions();
+
+        private static Resolution[] GetResolutions()
+        {
+            Resolution[] resolutions = Screen.resolutions;
+
+            int resolutionsLength = resolutions.Length;
+
+            for (int i = resolutionsLength - 1; i >= 0; i--)
+            {
+                if (resolutions[i].width > 1920) continue;
+                defaultIndex = resolutionsLength - 1 - i;
+                break;
+            }
+
+            return resolutions;
+        }
+
+        public static int GetDefaultIndex() =>
```

Comment:
would it be easier just to make public get property?
`public static int DefaultIndex {public get; private set;}`